next.config.js

```diff
@@ -3,3 +3,16 @@ module.exports = {
     SITE_NAME: process.env.SITE_NAME
   }
 }
+module.exports = {
```

Comment:
This is incorrect, merge exports to a single object. This will and is fail(ing) in production.

Reply:
Okay, Bhaiya! Now I will take care of such things.